src/main/java/co/dapi/DapiRequest.java

```diff
@@ -58,6 +61,13 @@ private static Response doRequest(String bodyJson, String url, HashMap<String, S
             headersBuilder.set(header.getKey(), header.getValue());
         }
         headersBuilder.set("content-type", "application/json");
+        Package objPackage = co.dapi.DapiApp.class.getPackage();
+
+        String version = objPackage.getImplementationVersion() != null ? objPackage.getImplementationVersion() : LIBRARY_VERSION;
+        String artifactId = objPackage.getImplementationTitle() != null ? objPackage.getImplementationTitle() : LIBRARY;
+        headersBuilder.set("X-Dapi-Library-Version", version);
+        headersBuilder.set("X-Dapi-Library", artifactId);
```

Comment:
Use the defined static values directly.

Reply:
https://github.com/dapi-co/dapi-java/pull/21/commits/574895364b30eec5150ab5f69062a55c1f6f1e4a